app/helpers/foreman_wreckingball/hosts/status_managed_hosts_dashboard_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module ForemanWreckingball
+  module Hosts
+    module StatusManagedHostsDashboardHelper
+      def hostname_or_link_to(host)
```

Comment:
Isn't that what [link_to_if_authorized](https://github.com/theforeman/foreman/blob/07426f676f061ea57eb689ff4ebe83b3c8eba4b7/app/helpers/application_helper.rb#L121-L132) already does?